internal/helm/repository/oci_chart_repository.go

```diff
@@ -118,10 +119,15 @@ func NewOCIChartRepository(repositoryURL string, chartRepoOpts ...OCIChartReposi
 // to be a semver.Constraints compatible string. If version is empty, the latest
 // stable version will be returned and prerelease versions will be ignored.
 // adapted from https://github.com/helm/helm/blob/49819b4ef782e80b0c7f78c30bd76b51ebb56dc8/pkg/downloader/chart_downloader.go#L162
-func (r *OCIChartRepository) Get(name, ver string) (*repo.ChartVersion, error) {
```

Comment:
I would like to keep the pointer receivers if possible here. I'd like to avoid copying the value on each method call.